lib/components/cards/VeggieCard.dart

```diff
@@ -22,25 +22,21 @@ class VeggieCard extends StatelessWidget {
         );
       },
       child: Container(
-        decoration: $styles.containerStyles.rounded,
-        child: Padding(
-          padding: EdgeInsets.all($styles.padding.m),
-          child: Column(
-            children: [
-              VeggieImage(veggie: veggie, size: size),
-              Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
```

Comment:
Column inside of Column not needed with some alignment instead. Also reduce all the Padding widgets